### PR TITLE
CSRF token check for all post methods with valid userid

### PR DESCRIPTION
**Pull request by Astthor**
Currently we only have one form for authenticated users.
The csrf token is an encrypted string using the AES crypto algorithm, encrypting only the userId.
The CSRFGuard is on an application level, but only activates if the request is a POST request and it has a valid session and includes a userId.
So in case we create a new form in the future in the application, it will break during development if we forget to fetch a CSRF token from the user endpoint and add it to the form.
Test cases weren't extensive so there might be some edge cases that I didn't pick up on.

### CODE REVIEW

File: public/components/post/createPost.html [hunk 1]
```diff
@@ -27,4 +27,15 @@
...
-</div>
\ No newline at end of file
+</div>
+
+<script>
+
+	fetch('/api/csrf-token')
+		.then((res) => res.json())
+		.then((userInfo) => {
+				console.log("info: " + JSON.stringify(userInfo));
```
Comment (ithai5) [inline public/components/post/createPost.html]: remove it before you merging

File: src/routes/some.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (ithai5) [inline src/routes/some.json]: what is it?

**Review by mstrucka — APPROVED** [review]
